Changelog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Version 3.0.0 - Feature release
+
+ - Add more supported Python versions. This plugin can now 3.10 and 3.11
```

Comment:
You are missing a word

Reply:
note: this PR is pending work still, waiting to see if we can drop 2.7 or not before investing more time on it